### ISSUE TITLE
CC2538 Router (Repeater Range extender)

### ISSUE THREAD

**Issue by Xsev1** [Koenkk/zigbee2mqtt#4774]
@Koenkk 
Please add the following device to herdsman converter.
## Information about the device + link
Zigbee CC2538 Router (repeater)
https://github.com/jethome-ru/zigbee-firmware/tree/master/ti/router/cc2538_cc2592

## data/database.db entry of the device
DB:
Link Quality

model:cc2538.router.v1
type:Router
nwk:49795
manuf id:0
manufacturer:jethome
power:Mains (single phase)
hard version:1
zcl version:1
date code:20200522
interviewed:true
configured:false
endpoint:1
profile:260
input clusters:genBasic (0)
genIdentify (3)
output clusters:genBasic (0)

More Information:

`Endpoint: 1
Profile: Home Automation (0x0104)
Application Device: Unknown (0x0100)
Input Cluster Count: 2
Input Cluster List
    Input Cluster: Basic (0x0000)
    
        Attribute: ZCL Version (0x0000)
        Data Type: 8-Bit Unsigned Integer (0x20)
        Uint8: 1 (0x01)
            
        Attribute: HW Version (0x0003)
        Data Type: 8-Bit Unsigned Integer (0x20)
        Uint8: 1 (0x01)
        
        Attribute: Manufacturer Name (0x0004)
        Data Type: Character String (0x42)
        String: jethome
    
        Attribute: Model Identifier (0x0005)
        Data Type: Character String (0x42)
        String: cc2538.router.v1
   
        Attribute: Date Code (0x0006)
        Data Type: Character String (0x42)
        String: 20200520 

    Input Cluster: Identify (0x0003)
Output Cluster Count: 1
Output Cluster List
    Output Cluster: Basic (0x0000)`

**Comment by Koenkk** [comment]
Please follow https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html and create a pr for https://github.com/Koenkk/zigbee-herdsman-converters

**Comment by dzikus** [comment]
> Please follow https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html and create a pr for https://github.com/Koenkk/zigbee-herdsman-converters

I did exactly what is described in the link above.

ad.1 I paired my cc2538+cc2592 router.
ad.2 I added entry to node_modules/zigbee-herdsman-converters/devices.js
ad.3 Nothing to do, router do not need converters.
ad.4 ignored as optional
ad.5 ignored as optional
ad.6 done

So how can we follow your guide and have cc2538 as router if in your instruction there is no merge of our code with yours and you do not want to add id by yourself.

Be consistent and add a decent manual as you refer people to it, dismissing them!
